pkg/catalogv2/content/content.go

```diff
@@ -196,6 +226,15 @@ func deepCopyIndex(src *repo.IndexFile) *repo.IndexFile {
 	return &deepcopy
 }
 
+// filterReleases - given:
+//   - IndexFile
+//   - the Kubernetes version
+//   - flag indicating whether to skip filtering releases
+//
+// filters out any chart versions that do not match the Rancher and Kubernetes versions, if specified in the chart's annotations.
+// If settings.IsRelease() is true, filters out any chart versions that are not compatible with the server's version.
```

Comment:
It is not clear when it does uncompatible. 

Can you add documentation here or in the code blocks ?